Common/Statistics/AlgorithmPerformance.cs

```diff
@@ -0,0 +1,74 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+
+namespace QuantConnect.Statistics
+{
+    /// <summary>
+    /// The AlgorithmPerformance is a wrapper class for TradeStatistics and PortfolioStatistics
+    /// </summary>
+    public class AlgorithmPerformance
+    {
+        /// <summary>
+        /// The algorithm statistics on closed trades
+        /// </summary>
+        public TradeStatistics TradeStatistics { get; private set; }
+
+        /// <summary>
+        /// The algorithm statistics on portfolio
+        /// </summary>
+        public PortfolioStatistics PortfolioStatistics { get; private set; }
+
+        /// <summary>
+        /// The list of closed trades
+        /// </summary>
+        public List<Trade> ClosedTrades { get; private set; }
+
+        /// <summary>
+        /// Initializes a new instance of the TradeStatistics class
```

Comment:
Looks like a lingering previous comment from a rename, recently I've been using:
```xml
 <see cref="AlgorithmPerformance"/>
```
in the comments so refactoring tools pick up the renames for me